spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/AveragePooling1D.scala

```diff
@@ -25,6 +25,19 @@ import com.intel.analytics.bigdl.utils.Shape
 
 import scala.reflect.ClassTag
 
+/**
+ * Applies average pooling operation for temporal data.
+ * The input of this layer should be 3D.
+ *
+ * When you use this layer as the first layer of a model, you need to provide the argument
+ * inputShape (a Single Shape, does not include the batch dimension).
+ *
+ * @param poolLength Size of the region to which average pooling is applied. Integer. Default is 2.
+ * @param stride Factor by which to downscale. Positive integer, or -1. 2 will halve the input.
+ *               If -1, it will default to poolLength. Default is -1.
```

Comment:
Default is -1, and in this case it will
 + *                be equal to poolSize.